lib.rs

```diff
@@ -461,18 +446,12 @@ impl Drain for AsyncCore {
             .serialize(record, &mut ser)
```

Comment:
Maybe I'm missing something.. But isn't this `serialize()` now happening twice? Once here, and once below in the `AsyncRecord::from()`?

@gnunicorn #dpc

Reply:
I think you're right.

Reply:
@gnunicorn @njam - anyone willing to prepare a PR to reuse the serialization part?

Reply:
I think it's just about removing the unnecessary call to serialization here?

-> https://github.com/slog-rs/async/pull/25

Reply:
btw thanks for exposing `AsyncRecord`, it's indeed useful!